packages/fiori-freestyle-writer/README.md

```diff
@@ -1,7 +1,8 @@
 # @sap-ux/fiori-freestyle-writer
 
-Easy to use Fiori Freestyle project files writer for use within Yeoman generator and other prompting libraries. 
+Writer module allowing to generate custom SAPUI5 applications with different templates.
 
+:warning: **All templates except the Basic template have been deprectated** and will be removed at some point in time. Please use the Custom page SAP Fiori template (`@sap-ux/fiori-elements-writer`)as an alternative.
```

Comment:
Please update the following:
- Spell check `deprectated`
- Change `at some point in time` to `by the end of 2023`

Reply:
Type fixed, text changed to match the message in the Fiori tools word by word. No, I won't add the specific date.

Reply:
thanks. `in a future release` sounds better :-)